bazel.1

```diff
@@ -0,0 +1,99 @@
+.TH BAZEL 1 "August 2020"
+.SH NAME
+bazel, bazel-3 \- tool to automate software builds and tests
+.SH SYNOPSIS
+.B bazel
+\fI\,<command> <options> \/\fR...
+.SH DESCRIPTION
+.IP
+[bazel release 3.4.0]
```

Comment:
This should be autogenerated, right?

Reply:
It would be nice if it were but I'm not sure how you would do that. We could also put something else here for a description. This was just generic. I was going for something simple that could be expanded upon later. :)

Could use "Build and test software of any size, quickly and reliably" instead.

Reply:
@olekw will you work on this?